rfc-brief.Rmd

```diff
@@ -0,0 +1,225 @@
+
+```{r fig-required, fig.cap="Survey responses for whether depositing raw data should be required for different assays", echo=F}
+
+q_required <- grep("should.it.be.required.to.deposit", names(survey), val = T)
+r_required <- survey %>% 
+  select(all_of(q_required)) %>%
+  pivot_longer(cols = all_of(q_required), names_to = "assay", values_to = "response") %>%
+  mutate(
+    assay = regmatches(assay, regexpr("^.+(?=..should)", assay, perl = T)),
+    response = recode(na_if(response, ""), .missing = "Unspecified")) %>% 
+  count(assay, response)
+
+defaultPlot(r_required, "Required")
+
+# could do a plot using percentages, 
+# but could seem disingenuous to "hide" that there were only 20 responses
+```
+
+Figure \@ref(fig:fig-interest-reanalyze) summarizes responses to the question "Have you ever re-analyzed this type of data, or wanted to, if the right dataset existed?" for the different assay or different type categories.
+The highest interest was for structured clinical data, which also garnered the strongest consensus that it should be shared (Fig \@ref(fig:fig-required)).
+
+
+```{r fig-interest-reanalyze, fig.cap="Survey responses for previous experience or interest in re-analyzing data for different assays", echo=F}
+
+q_interest <- grep("have.you.ever.re.analyzed", names(survey), val = T)
+r_interest <- survey %>% 
+  select(all_of(q_interest)) %>%
+  pivot_longer(cols = all_of(q_interest), names_to = "assay", values_to = "response") %>%
+  mutate(
+    assay = regmatches(assay, regexpr("^.+(?=..have)", assay, perl = T)),
+    response = recode(na_if(response, ""), .missing = "Unspecified")) %>% 
+  count(assay, response)
+
+defaultPlot(r_interest, "Required")
+
+```
+
+Figure \@ref(fig:fig-other-required) summarizes responses to the question "Have you previously been required to share a complete raw dataset by a funder or journal?"
+The responses suggest that funder or journal sharing requirements for these assays or data types are not common, except for perhaps structured clinical data and western blots.
+Compared to ratings in Fig \@ref(fig:fig-required), the results here also suggest that the community is willing to go beyond the "baseline" requirements of funders or journals.
+
+```{r fig-other-required, fig.cap="Survey responses for previous requirements to share raw data by a funder or journal", echo=F}
+
+q_other_required <- grep("by.a.funder.or.journal", names(survey), val = T)
+r_other_required <- survey %>% 
+  select(all_of(q_other_required)) %>%
+  pivot_longer(cols = all_of(q_other_required), names_to = "assay", values_to = "response") %>%
+  mutate(
+    assay = regmatches(assay, regexpr("^.+(?=..Have)", assay, perl = T)),
+    response = recode(na_if(response, ""), .missing = "Unspecified")) %>% 
+  count(assay, response)
+
+defaultPlot(r_other_required, "Required")
+
+```
+
```

Comment:
My suggestion would be to add a few other sections:
1. Methodology - describing what was done, how it was analyzed
2. discussion : highlight uses of reusable data e.g. secondary research, highlight key points from the survey, highlight possibilities going forward.
3. conclusion and thanks to the responders

Reply:
Let's merge and add this as an issue :)